aocd/models.py

```diff
@@ -524,3 +524,13 @@ def _parse_duration(s):
         return timedelta(hours=24)
     h, m, s = [int(x) for x in s.split(":")]
     return timedelta(hours=h, minutes=m, seconds=s)
+
+
+def load_users():
```

Comment:
Doesn't really belong in `models.py`, since it's a helper function and not a model.  Suggest to leave this part in `runner.py` directly, and probably give it an underscore prefix so that it's considered an implementation detail

Reply:
Sure. I put it in there as default_user was in there as well. 
Will push that change now.